StarterProject/pageobjects/page.js

```diff
@@ -0,0 +1,10 @@
+export default class Page {
+    constructor() {
+        
+    }
+    open(path) {
+        this.path = path ;
+        this.browser=browser;
+        browser.url(path);
+    }
```

Comment:
I'm not sure this `Page` class is necessary -- right now all this does is call `browser.url(path)`, which a test can call just fine 👍

Reply:
> right now

Good point, one day we might want to reduce duplicate code... in that case my recommendation would be to favor an abstract class.